CMakeLists.txt

```diff
@@ -68,7 +68,9 @@ if (WITH_GFLAGS)
     add_definitions(-DS2_USE_GFLAGS)
 endif()
 
-find_package(absl REQUIRED)
+if (NOT TARGET absl::base)
```

Comment:
If I just saw this code without having seen the PR, I would have no idea what it's doing or why.

Could you add a comment?

Reply:
I will add

Reply:
https://github.com/google/s2geometry/pull/286